pg_search/src/postgres/build_parallel.rs

```diff
@@ -250,6 +249,22 @@ impl WorkerBuildState {
             per_row_context: PgMemoryContexts::new("pg_search ambuild context"),
         })
     }
+
+    fn target_docs_per_segment(indexrel: &PgRelation) -> usize {
+        let desired_segment_count = std::thread::available_parallelism()
+            .expect("your computer should have at least one core");
+        let reltuples = indexrel
+            .heap_relation()
+            .unwrap()
+            .reltuples()
+            .unwrap_or_default();
+
+        if reltuples <= 0.0 {
+            pgrx::warning!("Unable to estimate the size of the table, run ANALYZE");
```

Comment:
I think we need to try and fake a reasonable number here.  

One way to do this would be to get the first page of the heap and count the number of items on it.  Then multiply that by the number of pages in the heap.  Otherwise, this function will return zero.